tests/multiparser_vue/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -68,28 +54,25 @@ p { font-size : 2em ; text-align : center ; }
 
   </style >
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-
-<template>
-
-  <h1>{{greeting}} world</h1>
-</template>
+<template >
+  <h1 >{{greeting}}     world</h1 >
+  <script>kikoo ( ) </script>
```

Comment:
Why not format non-toplevel scripts? What's the concern?

Reply:
I don’t want to add a lot of heuristics to figure out the correct indentation level